src/app/tvshows/tvshows.module.ts

```diff
@@ -14,6 +15,7 @@ import {
   imports: [
     CommonModule,
     RouterModule,
+    MaterialUiModule,
```

Comment:
Why not include this in a shared module rather then include it in each feature module? Isn't it likely to be used in all feature modules?